internal/info/infobuffer.go

```diff
@@ -160,6 +159,7 @@ func (i *InfoBuf) DonePrompt(canceled bool) {
 						break
 					}
 				}
+				i.PromptCallback(resp, false)
```

Comment:
nit: add empty line above this line?

Reply:
Done

Reply:
Please remove spaces from this empty line.

Reply:
Done